### DISCUSSION TITLE
Skipping identity verification on registration of TOTP

### DISCUSSION THREAD

**Discussion by Wouter1702** [authelia/authelia#8043] [Q&A] [answered]
Hello community,

Is there a way to skip the identity verification when trying to register a device for TOTP?
Now the system is putting the verification code in the notification.txt file in my authelia container.
I don't have a public smtp service available in my environment that can sent emails with my own email domain in the from address.
I use google work space (free version) for my email. Is there a way to setup authelia to sent smtp notifications without authentication problems to the google network.

I know skipping verification this is not secure, but this is for a home use setup.

Wouter.

**Comment by Crowley723** [comment, marked as answer]
I would suggest you look into the Google smtp relay. You can have authelia send emails using a Gmail account.
[Google SMTP](https://support.google.com/a/answer/176600?hl=en)
